src/main/java/net/sf/jabref/importer/fetcher/ACMPortalFetcher.java

```diff
@@ -60,6 +45,70 @@
 
 public class ACMPortalFetcher implements PreviewEntryFetcher {
 
+    private static final Pattern TITLE_PATTERN = Pattern.compile("<a href=.*?\">([^<]*)</a>");
+    private static final Pattern SOURCE_PATTERN = Pattern.compile("<span style=\"padding-left:10px\">([^<]*)</span>");
+    private static final Pattern ID_PATTERN = Pattern.compile("citation.cfm\\?id=(\\d+)&.*");
+
+    private int piv;
+
+    private static final String NEXT_ENTRY_PATTERN = "<div class=\"numbering\">";
+    private static final String AUTHOR_MARKER = "<div class=\"authors\">";
+    private static final String SOURCE_MARKER = "<div class=\"source\">";
+    private static final String END_ENTRY_PATTERN = "<br clear=\"all\" />";
+    private static final String SEARCH_URL_PART = "results.cfm?query=";
+    private static final String SEARCH_URL_PART_II = "&dl=";
+    private static final String END_URL = "&coll=Portal&short=0";//&start=";
+
```

Comment:
Order variables. ALL patterns together, then other static final variables and then empty line and variable piv